zqd/api/api.go

```diff
@@ -13,9 +13,10 @@ import (
 )
 
 type Error struct {
-	Type    string      `json:"type"`
-	Message string      `json:"error"`
-	Info    interface{} `json:"info,omitempty"`
+	Type      string      `json:"type"`
+	ErrorType string      `json:"error_type"`
```

Comment:
This could be useful, but I don't see it utilized in the PR?

Reply:
ooops... that's an omission that i've fixed in a follow-on commit. the purpose of this field is to hold the "Kind" string which was previously being written into Type. (This field could also be called Kind instead of ErrorType btw... happy to change if any strong preference).

Reply:
Ok! My naming preference is `Kind`, then.